wye/templates/workshops/workshop_detail.html

```diff
@@ -51,7 +51,11 @@ <h1 class="no-space">Workshop Details</h1>
                     <li class="list-group-item">
                         <div class="row">
                             <div class="col-xs-4"><strong>Instructors</strong></div>
```

Comment:
I guess problem is failing to create workshop.

File "/home/travis/build/pythonindia/wye/wye/workshops/views.py", line 163, in workshop_create

    'email_messages/workshop/create_workshop/message.html').render(context)

  File "/home/travis/virtualenv/python3.5.2/lib/python3.5/site-packages/django/template/backends/django.py", line 74, in render

    return self.template.render(context)

  File "/home/travis/virtualenv/python3.5.2/lib/python3.5/site-packages/django/template/base.py", line 210, in render

    return self._render(context)

  File "/home/travis/virtualenv/python3.5.2/lib/python3.5/site-packages/django/test/utils.py", line 96, in instrumented_test_render

    return self.nodelist.render(context)

  File "/home/travis/virtualenv/python3.5.2/lib/python3.5/site-packages/django/template/base.py", line 905, in render

    bit = self.render_node(node, context)

  File "/home/travis/virtualenv/python3.5.2/lib/python3.5/site-packages/django/template/base.py", line 919, in render_node

    return node.render(context)

  File "/home/travis/virtualenv/python3.5.2/lib/python3.5/site-packages/django/template/base.py", line 959, in render

    output = self.filter_expression.resolve(context)

  File "/home/travis/virtualenv/python3.5.2/lib/python3.5/site-packages/django/template/base.py", line 656, in resolve

    return string_if_invalid % self.var

  File "/home/travis/virtualenv/python3.5.2/lib/python3.5/site-packages/pytest_django/plugin.py", line 483, in __mod__

    pytest.fail(msg, pytrace=False)

  File "/home/travis/virtualenv/python3.5.2/lib/python3.5/site-packages/_pytest/runner.py", line 486, in fail

    raise Failed(msg=msg, pytrace=pytrace)

Failed: Undefined template variable 'workshop_topic' in 'email_messages/workshop/create_workshop/message.html'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):

  File "/home/travis/virtualenv/python3.5.2/lib/python3.5/site-packages/django/template/base.py", line 819, in _resolve_lookup

    current = current[bit]

  File "/home/travis/virtualenv/python3.5.2/lib/python3.5/site-packages/django/template/context.py", line 71, in __getitem__

    raise KeyError(key)

KeyError: 'SITE_INFO'